### DISCUSSION TITLE
Denial Student Developer Pack (German Student)

### DISCUSSION THREAD

**Discussion by siliconsniffer** [community/community#52844] [GitHub Education]
[please delete this post, spam]

**Comment by dafeist-github** [comment]
Same issue here. I tried to use OBS Virtual Cam since my front camera is way too bad to even make the text readable, and it still denys me. The insanely frustrating thing is that with the OBS-Cam, it instantly rejects me WITHOUT A REASON. It just tells me that "Unfortunately, we were unable to verify your academic affiliation based on the information you submitted" without any reason under it. The text is very clearly readable

**Comment by samus-aran** [comment]
Hi @lynx197 👋 

First off I want to thank you for sharing your experiences and that we absolutely understand your frustration. We are trying our best to continually improve our system to be as easy as possible for legitimate students everywhere to verify their academic status, while also ensuring that bad actors impersonating students don't jeopardise our ability to provide academic benefits of value.

As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the student applying and we thank you for being patient.

I suspect that your latest application didn't provide a very specific rejection reason. When students receive this rejection reason (or lack of) we would usually suggest that you provide additional pieces of evidences. Ensure you select the most appropriate evidence type in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

Hope this helps! Let us know how you get along 👍
